frontend/src/metabase/dashboard/containers/DashboardHeader.jsx

```diff
@@ -8,22 +9,38 @@ import Header from "metabase/components/Header";
 import Icon from "metabase/components/Icon";
 import Tooltip from "metabase/components/Tooltip";
 
-import { getDashboardActions } from "./DashboardActions";
+import Bookmark from "metabase/entities/bookmarks";
+import { getDashboardActions } from "metabase/dashboard/components/DashboardActions";
 import { DashboardHeaderButton } from "./DashboardHeader.styled";
 
-import ParametersPopover from "./ParametersPopover";
+import ParametersPopover from "metabase/dashboard/components/ParametersPopover";
 import Popover from "metabase/components/Popover";
 import PopoverWithTrigger from "metabase/components/PopoverWithTrigger";
+import { getIsBookmarked } from "./selectors";
 
 import cx from "classnames";
 
 import { Link } from "react-router";
 
+const mapStateToProps = (state, props) => {
+  return {
+    isBookmarked: getIsBookmarked(state, props),
```

Comment:
Could you add a similar selector for collections https://github.com/metabase/metabase/blob/946978d8677715862ae6ef93a0ab44c89e2de038/frontend/src/metabase/collections/containers/CollectionContent.jsx#L79